db/seeds.rb

```diff
@@ -7,3 +7,8 @@
 FactoryBot.create_list(:company, 50)
 
 FactoryBot.create_list(:billing_contract, 100)
+
+BillingContract.all.each do |billing_contract| 
```

Comment:
seed more multiple bill and payment for contract so that there is some monthly history